packages/indexer/lib/index.js

```diff
@@ -197,11 +199,28 @@ class CellCollector {
     if (!lock && typeof type !== "object") {
       throw new Error("Either lock or type script must be provided!");
     }
-    if (lock) {
+    // Wrap the plain `Script` to `ScriptWrapper`.
+    if (lock && !lock.script) {
       validators.ValidateScript(lock);
+      this.lock = { script: lock, argsLen: argsLen };
```

Comment:
Missing `ioType` or it has default value ?

Reply:
A code reuse issue, `ioType` in  [line 216](https://github.com/nervosnetwork/lumos/pull/106/files/f714e671fd4ffb34362cf4bff27e95c2fa9388dd#diff-bf9cee66d42f8a2eb8721be2b5937cbeR216) is supposed to be ignored. Fixed.